hw1/hw1.rst

```diff
@@ -32,42 +32,46 @@ If you have any feedback about errors, big or small, please let me know!
 .. _gem5: http://gem5.org/Main_Page
 .. _tutorial: http://learning.gem5.org/book/
 
-Instructions
-------------
+Assignment
+----------
 
 This assignment must be done alone.
 
-Step 1: gem5 Tutorial, Part I
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Step 1: Compile gem5
+~~~~~~~~~~~~~~~~~~~~
 
-Go through the Introduction and Part I of the `gem5 tutorial`__.
-You will go through the following topics:
+Go through the Introduction and Building gem5 pages of the `gem5 tutorial`__.
```

Comment:
We can do this now, in a later change, or I can do it next time I teach this, but I'd like rename tutorial->book everywhere.

Reply:
Resolved.